packages/ui/src/server/configs/e2e/local.ts

```diff
@@ -43,6 +43,7 @@ const e2eConfig: Partial<AppConfig> = {
 
         reUnipikaAllowTaggedSources: ['^https://yastatic\\.net/'],
         hideReferrerUrl: 'https://h.yandex-team.ru/',
+        reUseEffectiveAclForPath: '//sys/access_control_object_namespaces[^/+]{0,}',
```

Comment:
Editing of the file is a good reason to edit some files from `packages/UI/tests`.